browser/ui/webui/brave_federated/federated_internals_page_handler.cc

```diff
@@ -27,32 +31,40 @@ FederatedInternalsPageHandler::FederatedInternalsPageHandler(
               profile)
               ->GetDataStoreService()) {}
 
-FederatedInternalsPageHandler::~FederatedInternalsPageHandler() {}
+FederatedInternalsPageHandler::~FederatedInternalsPageHandler() = default;
 
-void FederatedInternalsPageHandler::GetAdStoreInfo() {
-  if (!data_store_service_)
+void FederatedInternalsPageHandler::UpdateDataStoresInfo() {
+  if (!data_store_service_) {
+    return;
+  }
+  brave_federated::AsyncDataStore* notification_ad_data_store =
+      data_store_service_->GetDataStore(
+          brave_federated::kNotificationAdTaskName);
```

Comment:
we should remove `brave_federated::` as we are already in the `brave_federated` namespace